uwac/libuwac/uwac-input.c

```diff
@@ -69,6 +73,10 @@ static struct wl_buffer* create_pointer_buffer(UwacSeat* seat, const void* src,
 	                              seat->pointer_image->width * 4, WL_SHM_FORMAT_ARGB8888);
 	wl_shm_pool_destroy(pool);
 
+	if (munmap(data, size) < 0)
```

Comment:
Sorry, did forget, here we only have `stdlib`, no `winpr` functions available.
use `fprintf`

Reply:
OK. Sure? I have not looked into the CMake files yet, but:
```
$ ldd uwac/libuwac/libuwac0.so.0.0.1 | grep local
	libfreerdp3.so.3 => /home/sascha/.local/lib/libfreerdp3.so.3 (0x00007f0883c3a000)
	libwinpr3.so.3 => /home/sascha/.local/lib/libwinpr3.so.3 (0x00007f0881c44000)
```

Reply:
yes. the linking against `libfreerdp` is not desired and will be removed, `libwinpr` is only linked privately to `libfreerdp` and not to `libuwac` (`objdump -x libuwac.so` explains it better)